Alertinator.php

```diff
@@ -174,3 +263,74 @@ protected function getTwilio() {
    }
 }
 
+interface alertLogger {
+   public function writeAlert($name, $status, $ts);
+   public function readAlerts($name);
+   public function resetAlerts($name);
+   public function isInAlert($name);
+}
+
+class fileLogger implements alertLogger {
+   
+   /**
+    * Write an alert to the logger.
+    *
+    * @param $name string  The key of the alert. Usually the name of the check fn.
+    * @param $status bool  0 = fail, 1 = success
+    * @param $ts int  Unix Timestamp of the event.
+   */
+   public function writeAlert($name, $status, $ts = FALSE) {
+      $ts = $ts ? $ts : time();
```

Comment:
`$ts = $ts ?? time();`

Reply:
I think you want `?:` no `??`. They have different behavior.